lib/devices_api/value_object_schema.ex

```diff
@@ -0,0 +1,16 @@
+defmodule DevicesApi.ValueObjectSchema do
+  @moduledoc """
+  Models a value object.
+  A value object is one that has no identity. It is mostly used as an embedded object.
+  """
```

Comment:
Não sei se você copiou direto, mas dê uma olhada no moduledoc com calma. Tem que pular um alinha entre a primeira frase e a próxima. Lembre da documentação oficial sobre documentaçÕes: https://hexdocs.pm/elixir/writing-documentation.html#recommendations

```suggestion
  @moduledoc """
  Models a value object.
  
  A value object is one that has no identity. It is mostly used as an embedded object.
  """
```